src/app/perspectiveapi-types.ts

```diff
@@ -100,7 +100,7 @@ export interface Score {
 // safety, since each interface accepts slightly different request and response
 // types.
 
-export interface GapiAnalyzeClient {
+export interface PerspectiveApiClient {
```

Comment:
Up to you, but might be worth adding Gapi to the name of this interface for clarity.